ignition/configuring-the-network.md

```diff
@@ -0,0 +1,41 @@
+# Configuring the Network #
+
+Configuring networkd with Ignition is a very straightforward task. Because
+Ignition runs before networkd starts, configuration is just a matter of writing
+the desired config to disk. The Ignition config has a specific section
+dedicated to this.
+
+## Static Networking ##
+
+In this example, the network interface with the name "eth0" will be given the
+IP address 10.0.1.7. A typical interface will need more configuration and can
+use all of the options of a [network unit][network].
+
+```json
+{
+	"networkd": {
+		"units": [
+			{
+				"name": "00-eth0.network",
+				"contents": "[Match]\nName=eth0\n\n[Network]\nAddress=10.0.1.7"
+			}
+		]
+	}
+}
+```
+
+This configuration will instruct Ignition to create a single network unit named
+"00-eth0.network" with the contents:
+
+```
+[Match]
+Name=eth0
+
+[Network]
+Address=10.0.1.7
+```
```

Comment:
I think our markdown renderer will represent that as a single line code block instead of a multi-line code block. Adding the extra new lines fixes this.